pkg/reconciler/v1alpha1/autoscaling/kpa_scaler.go

```diff
@@ -29,6 +30,11 @@ import (
 	clientset "github.com/knative/serving/pkg/client/clientset/versioned"
 )
 
+const (
+	// The period we wait after Active=False before actually scaling to zero.
+	gracePeriod = 60 * time.Second
```

Comment:
I believe this should be 2 minutes since that's the upper bound of Istio propagation delay.  @tcnghia, can you confirm?

Reply:
I am not sure if there is an upper bound to that.  60 seconds seems ok, we can adjust later if that isn't enough.

Reply:
Done